mkapp/mkapp_ota.sh

```diff
@@ -1,128 +1,39 @@
 #!/bin/bash
+set -x
```

Comment:
Probably remove this as it make the build a bit noisy

Reply:
....left over from debugging. tho a `set -e` might make sense actually.

Reply:
Yeah `set -e` would be better.